pkg/local_workflows/testdata/sarif-snyk-goof-ignores.json

```diff
@@ -2060,7 +2060,10 @@
                 "type": "fixExamples"
               }
             ],
-            "isAutofixable": false
+            "isAutofixable": false,
+            "snykPolicy/v1": {
+              "originalLevel": "warning"
+            }
```

Comment:
**issue (blocking):** The test data here does not consistently align with the updated schema we introduced here. Is that an intentional difference? If so what are we are verifying with these variations?
https://github.com/snyk/code-client-go/commit/c1be7d5b23feef0b95f4db25e866a3b2f7c81345#diff-be74a5b9c2e270fba071bb0defccfcb489a09c69b0a2b6aa8e5fe3ec8e5327c9R99-R103